### DISCUSSION TITLE
Question about colors codes

### DISCUSSION THREAD

**Discussion by kimllee** [Bash-it/bash-it#2179] [Q&A] [answered]
Hello,

I'd like to know where I could find a table colors codes for the differents themes please ?
For example, powerline theme.

Thank you.

**Comment by matchuek** [comment]
Hi @kimllee,
this is what you're looking for: https://github.com/Bash-it/bash-it/blob/master/themes/colors.theme.bash

kind regards

**Comment by davidpfarrell** [comment, marked as answer]
Greetings !

Themes with configurable colors most often store those colors in the themes `theme.bash` file.

For example, here's Powerline's:
* https://github.com/Bash-it/bash-it/blob/master/themes/powerline/powerline.theme.bash

You'll see the various color values mentioned throughout ...

To review what colors your terminal has configured for the various values, here's a copy of [another post I made on the subject](https://github.com/Bash-it/bash-it/issues/1864#issuecomment-813522033):

---

I googled `"print 256 colors terminal"` and found this page:

* https://askubuntu.com/questions/821157/print-a-256-color-test-pattern-in-the-terminal

The answer I went with uses this gist:
* [HaleTom/print256colours.sh](https://gist.github.com/HaleTom/89ffe32783f89f403bba96bd7bcd1263)

Here's what the script looks like in my terminal:
<img width="995" alt="Screen Shot 2021-04-05 at 10 27 59 AM" src="https://user-images.githubusercontent.com/45741407/113604138-b5894680-95f9-11eb-9412-2cb55ba45b00.png">

*NOTE: I have modified values 0-15 to represent the [Solarized Color Scheme](https://ethanschoonover.com/solarized/)

You *should* find that the color/numbers correspond to those used in the powerline themes.

Hope that helps - lemme know how it goes!
